translations/binaries/slps_strings_items_key.json

```
@@ -334,7 +334,7 @@
             "string": "堅い土地を掘り起こすのに使う道具。[LINE]\n先端の両端がくちばしのように[LINE]\nとがっている。[END]",
             "original_bytes_length": 79,
             "available_bytes_size": 79,
-            "translation": "단단한 땅을 파는 데 사용하는 도구입니다.[LINE]\n끝 부분이 부리처럼[LINE]\n뾰족합니다.[END]"
+            "translation": "단단한 땅을 파는 데 사용하는 도구입니다.[LINE]\n끝 부분이 부리처럼 뾰족합니다.[END]"
```

```suggestion
            "translation": "단단한 땅을 파는 데 사용하는 도구입니다.[LINE]\n끝 부분이 부리처럼 뾰족합니다.[LINE]\n[END]"
```

본문이 두 줄밖에 안 되는 경우에는 표기를 맞추기 위함인지, 비어있는 세번째 줄을 추가해 표기하는 것 같더라고요. 실제로 적용해보고 이질감이 없는지 확인해봐야겠습니다.